README.md

```diff
@@ -24,25 +24,20 @@ http://localhost:8118
 
 ## Request:
 
-`curl -X POST -H "Content-Type: application/json" http://localhost:8118/ingress/messages -d <payload>`
```

Comment:
move npm run test to another heading; and say run in diff session as these are integration test not unit

Reply:
done